File: Day_23/Q92.java
import java.util.*;

class Q92{
    static Scanner sc = new Scanner(System.in);

    //Function To Find Maximum Occurring Character
    static char max_occ_char(String str){
        HashMap<Character,Integer> freq = new HashMap<>();

        //Count Frequency
        for(int i = 0;i < str.length();i++){
            char ch = str.charAt(i);
            freq.put(ch,freq.getOrDefault(ch,0) + 1);
        }

        int maxi = 0;
        char ans = '\0';

        //Find Maximum Occurring Character
        for(char ch : freq.keySet()){
            if(freq.get(ch) > maxi){
                maxi = freq.get(ch);
                ans = ch;
            }
        }

        return ans;
    }

    public static void main(String args[]){
        String str = sc.nextLine();

        //Print Result
        System.out.println(max_occ_char(str));

        sc.close();
    }
}

File: Day_23/Q92.cpp
#include<bits/stdc++.h>
using namespace std;

//Function To Find Maximum Occurring Character
char max_occ_char(string str){
    unordered_map<char,int> freq;

    //Count Frequency
    for(int i = 0;i < str.length();i++){
        freq[str[i]]++;
    }

    int maxi = 0;
    char ans;

    //Find Maximum Occurring Character
    for(auto it : freq){
        if(it.second > maxi){
            maxi = it.second;
            ans = it.first;
        }
    }

    return ans;
}

int main(){
    string str;
    getline(cin,str);

    //Print Result
    cout << max_occ_char(str);

    return 0;
}
